waku/node.go

```diff
@@ -438,7 +438,7 @@ func printListeningAddresses(ctx context.Context, nodeOpts []node.WakuNodeOption
 		libp2pOpts = append(libp2pOpts, libp2p.AddrsFactory(addrFactory))
 	}
 
-	h, err := libp2p.New(ctx, libp2pOpts...)
+	h, err := libp2p.New(libp2pOpts...)
```

Comment:
This is the big API change with the new `libp2p`. Their recommendation in the Changelog was just to use `host.Close()` during shutdown, which go-waku already does